src/platform/utilities/sso/index.js

```diff
@@ -1,15 +1,25 @@
 import moment from 'moment';
 import environment from 'platform/utilities/environment';
+import { apiRequest } from 'platform/utilities/api';
 import localStorage from '../storage/localStorage';
-import { hasSession, hasSessionSSO } from '../../user/profile/utilities';
+import { hasSessionSSO } from '../../user/profile/utilities';
 import { login, logout } from 'platform/user/authentication/utilities';
 import mockKeepAlive from './mockKeepAliveSSO';
 import liveKeepAlive from './keepAliveSSO';
-import { getForceAuth } from './forceAuth';
+import { getLoginAttempted } from './loginAttempted';
 
 const keepAlive = environment.isLocalhost() ? mockKeepAlive : liveKeepAlive;
 const keepAliveThreshold = 5 * 60 * 1000; // 5 minutes, in milliseconds
 
+async function hasVAGovSession() {
+  try {
+    await apiRequest('/user', { method: 'HEAD' });
```

Comment:
I'm assuming that we're just testing for non-`4xx` errors here, which will go to the catch?

Reply:
I think anything other than `2XX` or `304` will raise an exception

Reply:
@U-DON I believe the `apiRequest` function returns a Promise, and if I await it will either return the payload or raise an exception.  Am I thinking about this correctly?

Reply:
Yup, that sounds right. Note that 401 does a redirect, but yes, anything other than 200s or 304 would throw.

Reply:
oh I was hoping a 401 would raise an error as well, maybe I can't use the `apiRequest` function here